Script Analysis Examples/Test Data Analysis/Failure_Pareto_Result_Analysis.ipynb

```diff
@@ -0,0 +1,529 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Failure Pareto for Test Results\n",
+    "This notebook creates a failure pareto report for test results. It ties into the **Test Monitor Service** for retrieving filtered test results, the **Notebook Execution Service** for running outside of Jupyterhub, and the **Test Monitor Reports page** at #testmonitor/reports for displaying results.\n",
+    "\n",
+    "The parameters and output use a schema recognized by the Test Monitor Reports page, which can be implemented by various report types. The Failure Pareto notebook produces data that is best shown in a pareto chart."
```

Comment:
Here is another reference to the reports page. Can you please scrub the document and make sure all of the text in the markdown sections and all the variables in the code sections make sense in the SLE context?

Reply:
Removed